pkg/asset/installconfig/platform.go

```diff
@@ -92,3 +101,7 @@ func (a *platform) queryUserForPlatform() (platform string, err error) {
 	}, &platform)
 	return
 }
+
+func (a *platform) CurrentPlatformName() string {
```

Comment:
nit: Platform is redundant in this function..

Reply:
this was marked resolved without comment or any changes...

Reply:
https://github.com/openshift/installer/pull/1555/commits/35968ea4cf54ac2b6e3b581f9e3224181f844ccd#diff-4d231774d44727d779db320e7b6c189eR39

Reply:
it's marked outdated